src/Config.h

```diff
@@ -10,6 +10,12 @@ class Config {
     void readFromFile(const string &file);
     void dumpCurrent(ofstream &fout);
     void dumpCurrent(ostream &out);
+    map<string, int> ArgmapInt;
```

Comment:
We can use a map with type `String -> Any` in plain C++ 17 and unify the 3 implementations.
See: https://en.cppreference.com/w/cpp/utility/any